drv/transceivers-server/src/main.rs

```diff
@@ -160,16 +187,87 @@ impl idl::InOrderTransceiversImpl for ServerImpl {
     }
 }
 
+impl NotificationHandler for ServerImpl {
+    fn current_notification_mask(&self) -> u32 {
+        TIMER_NOTIFICATION_MASK
+    }
+
+    fn handle_notification(&mut self, _bits: u32) {
```

Comment:
Add a comment explaining we only have one source of notification, so we don't need to check bits here.